### DISCUSSION TITLE
Can code-server be installed on termux?

### DISCUSSION THREAD

**Discussion by yuchen1204** [coder/code-server#5217] [Q&A]
Can code-server be installed on termux?

**Comment by lg-nineoneninethreenine** [comment]
Heck yeah - if you manage to install Node 14 successfully. That is the only caveat I can see, so if you figure it out, do share here! It would help a lot of community members to get this on Android.

**Comment by hfabio** [comment]
Wow, I can clearly see people using this with Samsung Dex for example (since we don't have a VSCODE for android/arm yet)

**Reply by lg-nineoneninethreenine** [reply]
Look, those just could be hacks, but I stalling this cleanly from code-server is better since it would be better compatible with your computer, you get a bunch of needed software, and you also get an IDE that just works. These VS Code apps take up memory. But in the readme of code-server itself, it is mentioned that it's lightweight, meaning light on both RAM and storage.
